### DISCUSSION TITLE
why does github pages is not showing anything of my web?

### DISCUSSION THREAD

**Discussion by corvo-the-lord** [community/community#55866] [New to GitHub]
### Select Topic Area

Question

### Body

here it is my repository
https://github.com/corvo-the-lord/GPT3

here is the link for the page 
https://corvo-the-lord.github.io/GPT3/

i have allready tried to change the name of public directory to doc and also take out the index.html from it bun nothing seems to work

**Comment by iamreiyn** [comment]
Hello @corvo-the-lord 

Check this detailed [guide](https://github.com/gitname/react-gh-pages) on how to publish your react app on GitHub pages, hope it helps!
